index-metagenomics.md

```diff
@@ -110,15 +110,15 @@ Both workflows consists of
 
 ### Analysis of metataxonomic data
 
-To analyze amplicon data, the **Mothur** and **QIIME** tool suites are available to ASaiM. We integrated the workflows described in tutorials of Mothur and QIIME websites, as example of amplicon data analyses as well as support for the training material. These workflows, as any workflows available in ASaiM, can be adapted for a specific analysis or used as subworkflows by the users.
+To analyze amplicon data, the **Mothur** and **QIIME** tool suites are available there. We integrated the workflows described in tutorials of Mothur and QIIME websites, as example of amplicon data analyses as well as support for the training material. These workflows, as any workflows available in ASaiM, can be adapted for a specific analysis or used as subworkflows by the users.
 
 ### Running as in EBI metagenomics
 
-The tools used in the EBI Metagenomics pipeline are also available in ASaiM. We integrate then also a [workflow](https://metagenomics.usegalaxy.eu/u/berenice/w/asaim-ebi-metagenomics-workflow-30) with the same steps as the [EBI Metagenomics pipeline (3.0)](https://www.ebi.ac.uk/metagenomics/pipelines/3.0).
+The tools used in the EBI Metagenomics pipeline are also available in ASaiM. We integrate them also a [workflow](https://metagenomics.usegalaxy.eu/u/berenice/w/asaim-ebi-metagenomics-workflow-30) with the same steps as the [EBI Metagenomics pipeline (3.0)](https://www.ebi.ac.uk/metagenomics/pipelines/3.0).
```

Comment:
This sentence is still wired. My proposal was different, you need at least to fix `We integrate them also a`